pynxtools_mpes/reader.py

```diff
@@ -185,15 +185,16 @@ def iterate_dictionary(dic, key_string):
     "angular_resolution": "RESOLUTION[angular_resolution]/resolution",
     "sample_temperature": "temperature_sensor/value",
     "drain_current": "drain_current_amperemeter/value",
+    "photon_energy": "energy",
 }
 
 REPLACE_NESTED = {
     "SAMPLE[sample]/chemical_formula": "SAMPLE[sample]/SUBSTANCE[substance]/molecular_formula_hill",
-    "source_TYPE[source]/Probe": "source_TYPE[source_probe]",
-    "source_TYPE[source]/Pump": "source_TYPE[source_pump]",
-    "beam_TYPE[beam]/Probe": "beam_TYPE[beam_probe]",
-    "beam_TYPE[beam]/Pump": "beam_TYPE[beam_pump]",
-    "sample_history": "sample_history/notes",
+    "sourceTYPE[source]/Probe": "sourceTYPE[source_probe]",
+    "sourceTYPE[source]/Pump": "sourceTYPE[source_pump]",
+    "beamTYPE[beam]/Probe": "beamTYPE[beam_probe]",
+    "beamTYPE[beam]/Pump": "beamTYPE[beam_pump]",
+    "sample_history": "history/notes/description",
```

Comment:
I required these additional changes to properly convert the eln data. Not sure though why the tests fail now.

Reply:
Jep, works now. I'll merge this then